src/sections/DonationDetailSection.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+import { Box, Heading, Text, Stack, Center, Grid } from '@chakra-ui/react';
+import { graphql } from 'gatsby';
+import Img from 'gatsby-image';
+import SectionHeader from 'src/components/SectionHeader';
+
+interface Props {
+  data: GatsbyTypes.DonationDetailFragment;
+}
+
+function DonationDetailSection({ data }: Props): JSX.Element {
+  const section = data.contentfulDonationDetailSection;
+
+  return (
+    <Box bg="gray.extralight">
+      <SectionHeader text="Where do donations go" textPosition="left" />
+      <Grid bg="charcoal" templateColumns={{ base: '1fr', lg: '1fr 1fr' }}>
```

Comment:
Should this be
```suggestion
      <Grid bg="charcoal" templateColumns={{ base: '1fr', md: '1fr 1fr' }}>
```

Reply:
When the breakpoint is at `md`, the text area gets narrow and long as such:
![image](https://user-images.githubusercontent.com/43834826/113728934-85f34080-96c4-11eb-9306-1c5c5ec78d88.png)

I think aesthetically, it looks better when the breakpoint it set at `lg`.

Reply:
Cool cool